autopts/ptsprojects/zephyr/tmap.py

```diff
@@ -102,12 +122,35 @@ def test_cases(ptses):
         TestFunc(stack.csip_init),
     ]
 
+    pre_conditions_2 = [
+        TestFunc(btp.core_reg_svc_mics),
+        TestFunc(btp.core_reg_svc_aics),
+        TestFunc(btp.core_reg_svc_vocs),
+        TestFunc(btp.core_reg_svc_gmcs),
+        TestFunc(btp.core_reg_svc_cas),
+        TestFunc(btp.core_reg_svc_micp),
+        TestFunc(stack.mics_init),
+        TestFunc(stack.aics_init),
+        TestFunc(stack.vocs_init),
+        TestFunc(stack.gmcs_init),
+        TestFunc(stack.micp_init),
+    ]
+
     adv_end = [
         TestFunc(btp.gap_set_extended_advertising_on),
         TestFunc(btp.gap_adv_ind_on, ad=advData),
     ]
 
     custom_test_cases = [
+        ZTestCase("TMAP", "TMAP/BMS/CTXT/BV-01-C", cmds=pre_conditions + pre_conditions_2 + [
+                TestFunc(adv_init, advData),
+                TestFunc(gap_set_uuid16_svc_data, advData, UUID.CAS,
+                          struct.pack('<B', CAPAnnouncement.GENERAL)),
+                TestFunc(gap_set_uuid16_svc_data, advData, UUID.ASCS,
+                          struct.pack('<BHHB', BAPAnnouncement.GENERAL,
+                Context.LIVE | Context.MEDIA, Context.LIVE, 0))] +
```

Comment:
Hmm, I assume it doesn't matter in the test, but above you set

```
    pts.set_pixit("TMAP", "TSPX_METADATA_SINK", "03020200")
    pts.set_pixit("TMAP", "TSPX_METADATA_SOURCE", "03020200")
```

but here you are setting metadata length to 0 to avoid advertising any metadata. Should we set the metadata here?